src/synthesizer/sed.py

```diff
@@ -1301,6 +1311,70 @@ def plot_spectra_as_rainbow(self, **kwargs):
         """
         return plot_spectra_as_rainbow(self, **kwargs)
 
+    # @accepts(velocity_fwhm=length/time, velocity_sigma=length/time)
+    def add_doppler_broadening(self, velocity_fwhm=None, velocity_sigma=None):
+        """
+        Add Doppler broadening to the spectra. This updates self.lnu to
```

Comment:
Trying to clean this up as I go but the first line of a doc string needs to be a single line sentence ending in a full stop.  Any further description can follow. (This factor in to how its parsed for the API).